Content/Library/build/build.fs

```diff
@@ -449,6 +457,15 @@ let generateCoverageReport _ =
 
     dotnet.reportgenerator id args
 
+let showCoverageReport _ =
+    failOnCIBuild ()
+    coverageReportDir </> "index.html"
+    |> Command.ShellCommand
+    |> CreateProcess.fromCommand
+    |> Proc.start
+    |> ignore
```

Comment:
I seem to remember this won't work on osx/linux.

Reply:
Oh that's awesome.

https://github.com/fsprojects/FAKE/blob/13e30330cae0597aed6154a95a06d21716b18de3/src/app/Fake.Core.Process/RawProc.fs#L163 looks like `Command.ShellCommand` will do it.